modules/ROOT/pages/site-structure.adoc

```diff
@@ -0,0 +1,57 @@
++
+** Individual API page variation: This default page variation is used when an API object is opened from the *API Card*, *API Carousel*, or *API Catalog* components.
++
+The API page variation includes the API name, icon, and description (xref:acm-lightning-components.adoc#api-header[API Header] component), its markdown documentation from Exchange (xref:acm-lightning-components.adoc#api-documentation-viewer[API Documentation Viewer] component), the API spec (xref:acm-lightning-components.adoc#api-console[API Console] component), links to download the API spec (xref:acm-lightning-components.adoc#spec-downloader[Spec Downloader] component), and the button to request access (xref:acm-lightning-components.adoc#api-access-requester[API Access Requester] component).
+** API group page variation: This page variation displays an API group when an API group object is opened from the *API Card*, *API Carousel*, or *API Catalog* components.
++
+The API group page variation includes the API group name, icon, and description (xref:acm-lightning-components.adoc#api-header[API Header] component), its markdown documentation from Exchange (xref:acm-lightning-components.adoc#api-documentation-viewer[API Documentation Viewer] component), an API carousel displaying the APIs that are part of this API group (xref:acm-lightning-components.adoc#api-carousel[API Carousel] component, with the *Show* setting configured for *Children*), and the button to request access to the API group (xref:acm-lightning-components.adoc#api-access-requester[API Access Requester] component).
++
+To configure this variation, assign a new audience to this page variation for the API group record type:
+
+. In Experience Builder, use the page selector at the top left to open the *Community API Details* page.
+. Click the gear icon next to the page name to open the page properties, and click the *Page variations* tab.
+. Click *New page variation*, select your preferred layout, click *Next*, and enter a name for the new variation, such as *API Group*.
++
+The new page variation displays.
+. Click the gear icon next to the page name to open the page properties, and click the *Page variations* tab.
+. On the line item for your new page variation, click the context menu, and under the *Audience* section, click *Assign*.
+. Select *New audience*, enter an audience name such as *API Group record type*, scroll down to open the *Advanced* section, and set these values:
+*** *Record type*: *CommunityApi*
+*** *Operator*: *Equals*
+*** *Value*: *APIGroup*
+. Click *Save* and *Assign*.
+. After the new audience has been assigned, use the components palette to add the desired components to your page variation.
```

Comment:
```suggestion
. After the new audience is assigned, use the components palette to customize your page variation.
```

Reply:
The following four list items should be added directly after their most recent peer item above: * API and API groups details page. The existing organization with a long detour is kind of disorienting.

Reply:
Updated.